docs/examples/MenuItem.js

```diff
@@ -0,0 +1,26 @@
+function onSelectAlert(eventKey, href, target) {
+  alert('Alert from menu item.\neventKey: "' + eventKey + '"\nhref: "' + href + '"');
+}
+
+function preventDefault() {}
+
+const MenuItems = (
+  <div className="clearfix">
+    <ul className="dropdown-menu open">
+      <MenuItem header>Header</MenuItem>
+      <MenuItem onSelect={preventDefault}>link</MenuItem>
```

Comment:
I'm not a fan of this, though I'm sure I'll have this cleaned up in the dropdown revisited branch so I guess we can live with it in the time being.